packages/less/Gruntfile.js

```diff
@@ -184,12 +183,6 @@ module.exports = function(grunt) {
     // Make the SauceLabs jobs
     ["all"].concat(browserTests).map(makeJob);
 
-    var path = require('path');
-
-    // Handle async / await in Rollup build for tests
-    const tsNodeRuntime = resolveBin.sync('ts-node');
```

Comment:
This is the root cause for the CI suspension, we don't need these code anymore since we upgrade ts-node.